fetch_depth_layer/src/depth_layer.cpp

```diff
@@ -84,6 +84,31 @@ void FetchDepthLayer::onInitialize()
   // Should skipped edge rays be used for clearing?
   private_nh.param("clear_with_skipped_rays", clear_with_skipped_rays_, false);
 
+  // How long should observations persist?
+  private_nh.param("observation_persistence", observation_keep_time, 0.0);
```

Comment:
Mainly just because ObstacleLayer does the same.  I think the param externally should stay "observation_persistence" to maintain consistency.  The variable internally is called what it is I guess just because that's what the ObservationBuffer header calls it.  I'm fine with changing it to observation_persistence also, though.